db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150416231106) do
+ActiveRecord::Schema.define(version: 20150511231229) do
```

Comment:
This schema.rb doesn't include the migrations to rename the table, or create the default project roles.